rclpy/rclpy/node.py

```diff
@@ -30,13 +33,32 @@
 
 class Node:
 
-    def __init__(self, handle):
+    def __init__(self, node_name, *, namespace=None):
         self.clients = []
-        self._handle = handle
+        self._handle = None
         self.publishers = []
         self.services = []
         self.subscriptions = []
         self.timers = []
+        self._default_callback_group = MutuallyExclusiveCallbackGroup()
+
+        namespace = namespace or ''
+        if not ok():
+            raise NotInitializedException('cannot create node')
+        try:
+            node_handle = _rclpy.rclpy_create_node(node_name, namespace)
+            self._handle = node_handle
```

Comment:
The temporary variable `node_handle` is needed. The result can directly be assigned to `self._handle`.

Reply:
8c4b1ddff392a444dd20491a48da22f71ee611ad